cmd/alerts/add_silence.go

```diff
@@ -0,0 +1,49 @@
+package alerts
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAddSilence() *cobra.Command {
+	return &cobra.Command{
+		Use:   "add-silence <cluster-id> <alertname1> <alertname2>... <comment>",
```

Comment:
For data like this, I'd prefer we use flags instead of multiple args. You can even declare a "slice" flag that takes multiple values for `alertname`

Reply:
Added 
```
cmd.Flags().StringSliceVar(&addSilenceCmd.alertID, "alertname", []string{}, "alertname (comma-separated)")
cmd.Flags().StringVarP(&addSilenceCmd.comment, "comment","c","","add comment about alertname")
```